docs-api/frontity-packages/collections-packages/components.md

```diff
@@ -144,6 +145,38 @@ const Link = ({
 export default connect(Link);
 ```
 
+#### The `link` processor
+
+Frontity provides a `link` processor. The `link` processor works with the `<html2react>` component and can automatically detect `<a>` tags in the page/post content and intelligently convert them into `<Link>` components.
+
+If the `href` attribute of the `<a>` tag is either:
+
+- a relative link, or
+- an absolute link on the same domain as the WordPress data source
+
+then the processor will convert the the `<a>` tag into a `<Link>` component.
+
+The `<Link>` component created by the processor will be modelled on the `<a>` tag and will have properties consistent with it's attributes - e.g. the `link` property of the `<Link>` component will be the same as the `href` attribute of `<a>` tag being replaced. The processor will also convert absolute links on the same domain to be relative links.
+
+If the `href` attribute of the `<a>` tag is an absolute link on a different domain from the WordPress data source, i.e. it is a link to an external site, then that tag will remain as is and will not be replaced or converted.
+
+In order for this to work two conditions must be met:
+
+1. The WordPress data source must be defined using `state.source.url`, it will not work if the data source is defined with the (now deprecated) `state.source.api`. See [here](https://api.frontity.org/frontity-packages/features-packages/wp-source#rest-api-settings) and [here](https://docs.frontity.org/guides/setting-url-wordpress-source-data) for more information on defining your data source URL.
```

Comment:
> [...] it will not work if the data source is defined with the (now deprecated) `state.source.api`.

I didn't expected that. Is this something you discovered while working on an example or tutorial? :thinking:

Reply:
No, I got it from @luisherranz's comment: https://github.com/frontity/api-reference/issues/17#issue-824789960

Reply:
Uhm, I feel that comment is outdated because it was published before we made `state.source.url` backwards compatible (see [/make-the-backend-url-a-global-setting/2381/56](https://community.frontity.org/t/make-the-backend-url-a-global-setting/2381/56)), so it should work with `state.source.api` (not recommended, though). Just following the guide about [Setting the URL of the WordPress data source](https://docs.frontity.org/guides/setting-url-wordpress-source-data) should be enough.

Reply:
I've removed the reference to that condition